front-end/src/components/Comment/Comment.tsx

```diff
@@ -41,22 +43,36 @@ interface Props{
 		Promise<ApolloQueryResult<DiscussionPostAndCommentsQuery>>
 }
 
-export const Comment = ({ className, comment, refetch } : Props) => {
+export const Comment = ({ className, comment, last, refetch } : Props) => {
 	const { author, content, created_at, id, updated_at } = comment;
+	const { hash } = useLocation();
+
+	useEffect(() => {
+		if (hash === `#${id}` || hash === '#end') {
+			setTimeout(
```

Comment:
Why using a setTimeout with 0 here?

Reply:
document.getElementById(id); will not find element unless checked in next tick of event loop. Will use useRef instead of getElementById